README.md

```diff
@@ -101,6 +104,30 @@ class Aircraft(models.Model):
 | `QuerySet.update()`           | No
 | `QuerySet.update_or_create()` | Yes (via `QuerySet.get_or_create()` and `Model.save()`)
 
+#### Auditing Special QuerySet Writes
+
+Auditing for the four "special" QuerySet methods that perform DB writes (labeled
+**No** in the table above) _can_ be enabled. This requires three extra usage
+details:
+
+1. Enable the feature by calling the audit decorator specifying
+   `@audit_fields(..., audit_special_queryset_writes=True)`.
+2. Configure the model class so its default manager is an instance of
+   `field_audit.models.AuditingManager`.
+3. All calls to the four "special" QuerySet write methods require an extra
+   `audit_action` keyword argument whose value is one of:
+   - `field_audit.models.AuditAction.AUDIT`
+   - `field_audit.models.AuditAction.IGNORE`
```

Comment:
Will a call to one of these "special" QuerySet methods raise an error if this argument isn't passed in? (My read is yes?)

Reply:
Correct. If you use this feature, you must always be explicit about what auditing behavior you want.